ground/src/main/java/com/google/android/ground/ui/map/gms/GmsExt.kt

```diff
@@ -33,5 +34,16 @@ object GmsExt {
 
   fun Bounds.center(): Coordinate = toGoogleMapsObject().center.toModelObject()
 
+  fun List<Geometry>.toBounds(): Bounds? {
+    val coordinates = this.map { it.vertices.first().coordinate }
```

Comment:
Is `vertices` is well-defined? e.g. what does it return for multipolygons with holes, for example?

Reply:
For polygon with holes, we only return the list of vertices of shell.